exporter/postgres/setup_pg12.sql

```diff
@@ -133,5 +133,141 @@ WHERE (ROUND(used/slots*100)::int) > p_percent;
 $function$;
 
 
+/*
+ * Table and functions for monitoring changes to pg_settings and pg_hba_file_rules system catalogs.
+ * Can't just do a raw check for the hash value since Prometheus only records numeric values for alerts
+ * Table allows recording of existing settings so it can be referred back to to see what changed
+ * If function returns 0, then NO settings have changed for either pg_settings or hba since last known valid state 
+ * If function returns 1, then just pg_settings have changed since last known valid state
+ * If function returns 2, then just hba has changed since last known valid state
+ * If function returns 3, then both pg_settings and hba have changed since last known valid state
+ */
+
+DROP TABLE IF EXISTS monitor.settings_checksum;
+DROP FUNCTION IF EXISTS monitor.settings_checksum();
+
+CREATE TABLE monitor.settings_checksum (
+    settings_hash text NOT NULL
+    , settings_string text NOT NULL
+    , hba_hash text NOT NULL
+    , hba_string text NOT NULL
+    , created_at timestamptz DEFAULT now() NOT NULL
```

Comment:
how about captured_on or recorded_on ?

Reply:
Open to suggestion. `created_at` is the standard column name I've countered for recording the timestamp of when that row was created.

Reply:
created_at works for me and jives w/ other tools i've seen/used